docs/pages/desktop-access/rbac.mdx

```diff
@@ -195,78 +195,6 @@ allow:
 
 ### Automatic User Creation
```

Comment:
Do we need to remove this header now that there's no text underneath it?

Reply:
We do, I removed it.